<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BibleWindow</class>
 <widget class="QWidget" name="BibleWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bible - KJV</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/aletheia/icons/resources/open_book_small.png</normaloff>:/aletheia/icons/resources/open_book_small.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="bookSelector">
       <property name="text">
        <string>Book</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="chapterSelector">
       <property name="text">
        <string>Chapter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="crossReferenceToggle">
       <property name="text">
        <string>Cross-references</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="textView">
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
     <property name="openLinks">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>textView</tabstop>
  <tabstop>bookSelector</tabstop>
  <tabstop>chapterSelector</tabstop>
  <tabstop>crossReferenceToggle</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>crossReferenceToggle</sender>
   <signal>toggled(bool)</signal>
   <receiver>BibleWindow</receiver>
   <slot>showCrossReferences(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showCrossReferences(bool)</slot>
 </slots>
</ui>
